apps/web/modules/signup-view.tsx

```diff
@@ -206,7 +206,7 @@ export default function Signup({
     }
   }, [redirectUrl]);
 
-  const [COOKIE_CONSENT, setCOOKIE_CONSENT] = useState(false);
+  const [COOKIE_CONSENT, setCOOKIE_CONSENT] = useState(false); // No need to be checked for user to proceed
```

Comment:
I think this is the first time I've ever set something named with this casing 😄

Reply:
Oh heeeel no 😆

Reply:
yeah lol, let me change it